todo.txt

```diff
@@ -20,5 +23,12 @@
 		-Thus for consistency, if a function CAN apply to a file, 
 		 it should execute once before exiting	
 
-***BUG FUNCTION IDEAS***
+
+Other random thoughts:
+
+-- I think the fill in the blank is a good idea, but it seems like it is a little overused... a few lines here and there would be cool to see but with my testing, it gets a little obnoxious.
```

Comment:
Do you mean that it applies to too many lines? Or that there are too many underscores in the line?

Reply:
I was thinking both on too many lines and on too many characters in the line.. it might be just me but I do like the feature its just I have gotten it to run on my tests lots of times...